packages/simple-form/tests/test-bundler.js

```diff
@@ -0,0 +1,14 @@
+import chaiEnzyme from 'chai-enzyme'
+chai.use(chaiEnzyme())
+
+const __karmaWebpackManifest__ = new Array() // eslint-disable-line
+const inManifest = path => ~__karmaWebpackManifest__.indexOf(path)
+
+const testsContext = require.context('../src', true, /\.spec\.js$/)
+
+const testsToRun = testsContext.keys().filter(inManifest)
+;(testsToRun.length ? testsToRun : testsContext.keys()).forEach(testsContext)
```

Comment:
typo? first character `;` on line 10

Reply:
not a typo. this is a strange file we copied from the startkit and enhanced it a bit. not 100% sure what every single line does to be honest :)